### ISSUE TITLE
some XML nodes from save file are not available in XSLT for Character Sheet creation

### ISSUE THREAD

**Issue by chummer5a** [chummer5a/chummer5a#119]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_
_Originally opened as https://github.com/chummer5a/chummer5/issues/45_

----

```

I have tried to create customized Character sheet editing 'Shadowrun 5  
Base.xslt' file.

I managed to "game notes" field, but "calendar/week" and "expenses/expense" XML 
nodes are not available. I have also displayed whole dataset, to verify if 
there is no name change but this XML tags are not there. 

Is there a way to access these nodes?
```

Original issue reported on code.google.com by `mikek...@gmail.com` on 11 Mar 2015 at 3:18

**Comment by chummer5a** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_

----

```
Hi again,

I have managed to add "expenses/expense" XML nodes by setting to 
<printexpenses>True</printexpenses> in default.xml

but still no luck with 'calendar"

```

Original comment by `mikek...@gmail.com` on 11 Mar 2015 at 3:27

**Comment by chummer5a** [comment]
Shouldn't be an issue. Will investigate.

**Comment by chummer5a** [comment]
Resolved in 5.159; Release blocked by completion of advanced lifestyles module.
